src/main/java/org/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -386,9 +386,17 @@ private void parseJabRefComment(Map<String, String> meta) {
             } catch (ParseException ex) {
                 parserResult.addException(ex);
             }
+        } else if (comment.startsWith(MetaData.META_FLAG_VERSION_010)) {
+            parseCommentToJson(comment);
```

Comment:
The result is not handled. I think, it needs to be handled. It needs to be passed to the `Metadata.parse` method somehow.